helm/prometheus-agent/charts/prometheus-agent/templates/_helpers.tpl

```diff
@@ -26,14 +26,40 @@ Common labels
 */}}
 {{- define "labels.common" -}}
 {{ include "labels.selector" . }}
-app.giantswarm.io/branch: {{ .Chart.Annotations.branch | replace "#" "-" | replace "/" "-" | replace "." "-" | trunc 63 | trimSuffix "-" | quote }}
-app.giantswarm.io/commit: {{ .Chart.Annotations.commit | quote }}
 app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
 app.kubernetes.io/version: {{ .Values.image.tag | default .Chart.AppVersion | quote }}
-app.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
+application.giantswarm.io/team: {{ index .Chart.Annotations "application.giantswarm.io/team" | quote }}
 helm.sh/chart: {{ include "chart" . | quote }}
 {{- end -}}
 
 {{- define "psp.name" -}}
 {{- include "name" . -}}-psp
 {{- end -}}
+
+{{- define "prometheus-agent.remote-write" -}}
+{{- if or .Values.global.remoteWrite .Values.remoteWrite -}}
```

Comment:
I think we should test the length of the array here, otherwise we could end up with invalid yaml when the array is empty.

Reply:
What do you mean by testing the length?  .Values.global.remoteWrite is falsy if the array is either nil or len(0) so if both are empty we would render the else which is remoteWrite: []

Reply:
Ok fine then. I though this would only test if the array is nil.